answerking_app/services/item_service.py

```diff
@@ -1,51 +1,55 @@
-from answerking_app.models.models import Item, Order, OrderLine
+from decimal import Decimal
+
+from django.db.models import QuerySet
+
+from answerking_app.models.models import Item, OrderLine
 from answerking_app.models.validation.serializers import ItemSerializer
 
 
-def get_all():
-    items = Item.objects.all()
+def get_all() -> QuerySet[Item] | list:
+    items: QuerySet[Item] = Item.objects.all()
 
     if not items:
```

Comment:
The return type of this function is kinda odd, do we need this part `if not items: return []` isn't an empty QuerySet the same thing? 🤔

Reply:
Pycharm complains when it is just  `QuerySet[Item]`. You get this message when hovering over the 'return []' - Expected type 'QuerySet[Item]', got 'list' instead

Reply:
Oh sorry you just mean the code in general?

Reply:
I mean:
```
def get_all() -> QuerySet[Item]:
   return Item.objects.all()
```

Can we refactor this function to this? I feel like it makes more sense.

Reply:
Yes I agree. I will apply the same to the other get_all functions

Reply:
This has been done